src/vdxf/keys.ts

```diff
@@ -673,4 +673,33 @@ export const APP_ENCRYPTION_REQUEST_DETAILS_VDXF_KEY: VDXFKeyInterface = {
     "namespace": "i5w5MuNik5NtLcYmNzcvaoixooEebB6MGV"
   },
   "vdxfid": "iEWvbyzxSQ3G2sdBTeU4DUBHhNFpd6rUFx"
+}
+
+export const DATA_DESCRIPTOR_RESPONSE_VDXF_KEY: VDXFKeyInterface = {
+  "hash160result": "2ff523bad4a35bfa3db954295acd192e1d805dd9",
+  "vdxfid": "iPHr5EDp6QGdbAJS8Rqu8293HyQrYjgRap",
+  "indexid": "xU7xY2etwiVJDLBTz7W46QfaKdRsSoos4g",
+  "qualifiedname": {
+    "namespace": "i5w5MuNik5NtLcYmNzcvaoixooEebB6MGV",
+    "name": "vrsc::response.generic.datadescriptor"
+  }
+}
+export const USER_DATA_REQUEST_DETAILS_VDXF_KEY: VDXFKeyInterface = {
```

Comment:
As discussed, do not use the word "details" in the class or key names unless it is required, e.g. in Provisioning Details because it isn't a request its just providing details about provisioning. This is not details, this is a request.